app/components/key_pair_generator_component.js

```diff
@@ -0,0 +1,40 @@
+import { trackEvent } from '@18f/identity-analytics';
+
+class KeyPairGeneratorElement extends HTMLElement {
+  connectedCallback() {
+    this.generateKeyPair().then(() => {
+      if (this.duration === undefined) {
+        this.duration = -1;
+      }
+      trackEvent('IdV: key pair generation', {
```

Comment:
Just to confirm: Do we care about the "Frontend" prefix of this event? Including it in [the frontend event mapping](https://github.com/18F/identity-idp/blob/1eda13c3e4ccc911c4aa013108cda1cc6a9c4b29/app/controllers/frontend_log_controller.rb#L7-L33) could avoid the prefix and ensure the event details are included in [the events handbook page](https://handbook.login.gov/articles/analytics-events.html).

Reply:
This event is only for this data collection and would go away once we are done so I think it's fine as is.